### DISCUSSION TITLE
Welcome to NetTopologySuite Discussions!

### DISCUSSION THREAD

**Discussion by airbreather** [NetTopologySuite/NetTopologySuite#458] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by airbreather** [comment]
^ that's the default template that GitHub gave me when I enabled Discussions for this repository.  I figured I'd leave it :grin:.

**Comment by DGuidi** [comment]
nice to have this space instead of using github issues used as a "forum"
